golem-worker-executor-base/src/worker.rs

```diff
@@ -86,6 +89,41 @@ pub struct Worker<Ctx: WorkerCtx> {
     oom_retry_config: RetryConfig,
 }
 
+impl<Ctx: WorkerCtx> Worker<Ctx> {
```

Comment:
Please don't open multiple `impl` sections for the same thing

Reply:
Did I? I searched for if there are duplicates. May be I missed out.

Reply:
Yes.

Reply:
Basically most of the file is a `impl<Ctx: WorkerCtx> Worker<Ctx> ` below.